feelslikemonday/app/src/main/java/com/example/feelslikemonday/DAO/UserDAO.java

```diff
@@ -49,16 +58,17 @@ public void onFailure(@NonNull Exception e) {
                 });
```

Comment:
Not sure if this really matters (i.e. will it crash or cause bugs) but in the lab we convert e to a String on the line 
 Log.w(TAG, "Error writing document", e); so that it would instead look like 
 Log.w(TAG, "Error writing document", e.toString());

Reply:
It works